fpm/recipes/gdal/recipe.rb

```diff
@@ -0,0 +1,20 @@
+class GDAL < FPM::Cookery::Recipe
+  description "Geospatial Data Abstraction Library"
+
+  name "gdal"
+  version "1.11.5"
+  license "MIT"
+  maintainer "GOV.UK <govuk-dev@digital.cabinet-office.gov.uk>"
+  homepage "http://www.gdal.org/"
+  source "http://download.osgeo.org/gdal/#{version}/gdal-#{version}.tar.gz"
```

Comment:
Can we use the `https://` URL?

Reply:
Good question.  The main GDAL site is only available on HTTP (at least, HTTPS has certificate issues), and the HTTP version of the file link is what's linked from their site, perhaps due to mixed content issues.

In any case, it looks like the file is accessible over HTTPS, so I've updated it.